fv3core/stencils/d_sw.py

```diff
@@ -18,14 +18,15 @@
 import fv3core.stencils.fvtp2d as fvtp2d
 import fv3core.stencils.fxadv as fxadv
 import fv3core.stencils.heatdiss as heatdiss
-import fv3core.stencils.vorticity_volumemean as vort_mean
 import fv3core.stencils.xtp_u as xtp_u
 import fv3core.stencils.ytp_v as ytp_v
 import fv3core.utils.corners as corners
 import fv3core.utils.global_constants as constants
 import fv3core.utils.gt4py_utils as utils
 from fv3core.decorators import gtstencil
 
+from ..utils.typing import FloatField
```

Comment:
I'd rather see all these or none follow relative imports.

Reply:
I'm fine with switching it.